dts/bindings/serial/zephyr,bt-uart.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2024 Croxel, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: UART over Bluetooth LE
+
+compatible: "zephyr,bt-uart"
```

Comment:
Bluetooth WG:

Agreement to call this `zephyr,nus-uart`

Reply:
Addressed.